manifest_schema.json

```diff
@@ -85,7 +85,7 @@
     },
     "app_id": {
       "type": "string",
-      "description": "A string that identifies the MiniApp univocally.",
+      "description": "A case sensitive string that identifies the MiniApp univocally.",
```

Comment:
See above about "univocally"

I would probably change this to read more like:

> A string containing a unique identifier for the MiniApp; case-sensitive.

Reply:
Changed as suggested. Thanks.